source/draw/DrawState.ooc

```diff
@@ -36,6 +36,7 @@ DrawState: cover {
 	_destinationNormalized := FloatBox2D new(0.0f, 0.0f, 1.0f, 1.0f)
 	_sourceNormalized := FloatBox2D new(0.0f, 0.0f, 1.0f, 1.0f)
 	_focalLengthNormalized := 0.0f // Relative to image width
+	flatTransform := FloatTransform2D identity // 2D scaling done after projection matrix but before w division
```

Comment:
I cannot think of a better name for now.
Using 3D terms like projection to describe a 2D operation does not sound abstract.
ImageTransform was used before to describe the source coordinates.